docs/DeveloperGuide.md

```diff
@@ -283,16 +312,53 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `GuestList` and the **Actor** is the `user`, unless specified otherwise)
+
+**Use case: Get list of all guests**
+
+**MSS**
+
+1.  User requests to list guests
+2.  GuestList shows a list of guests
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+**Use case: Add a guest**
+
+**MSS**
+
+1. User searches for guest 
+2. Guest is not found in list of guests 
+3. User requests to add a guest to the list
+4. GuestList adds the guest
+    
+    Use case ends.
+
+**Extensions**
+* 1a. The guest is found in the list of guests.
+
+    Use case ends.
+
+* 3a. The inputted fields are invalid
 
-**Use case: Delete a person**
+  * 3a1. GuestList shows an error message.
+  
+    Use case resumes at step 2.
+  
+**Use case: Delete a guest**
```

Comment:
Should we shift the use cases to be ordered based on what function comes first like our UG? ie Add -> Delete -> List -> Exit

Reply:
Yes, we should follow that order. Will change.